includes/class-coblocks-form.php

```diff
@@ -351,13 +352,32 @@ public function process_form_submission( $atts ) {
 
 		}
 
+		$spam_check = filter_input( INPUT_POST, 'coblocks-verify-email', FILTER_SANITIZE_STRING );
```

Comment:
@EvanHerman We should probably comment in the server side code here what is going on, so developers can clearly see this is our honeypot strategy and not a legit field.